### ISSUE TITLE
Why is CIPS your main competitor?

### ISSUE THREAD

**Issue by denkorzh** [BlingHe/PixelFolder#2]
Hi!

Thank you for sharing your work! However, I find the positioning of your work questionable to say the least. Despite the claim that CIPS is a major competitor due to repeated comparisons with this approach throughout the work, PixelFolder uses both the PixelShuffle operation and the 3x3 convolution kernels. This has nothing to do with independent pixel synthesis, which is the emphasis of the CIPS work. 

Furthermore, since your approach resembles more traditional architectures such as StyleGAN2, it would be great to show a comparison with this model on the FFHQ-1024 dataset, rather than limiting the resolution to 256x256. Do you have plans to share your results in hi-res in the future?

**Comment by BlingHe** [comment]
Thanks for your comment. 

Our work is greatly inspired by CIPS, so we use it as the main comparison method. In this paper, we also compare our method with other strong baselines such as StyleGAN and INR-GAN. Independent pixel synthesis is indeed an outstanding property of CIPS, which greatly simplifies the design of GANs. But we noticed that using only 1*1 conv would result in insufficient local imagery when generating large objects, so we resorted to pixel folding and 3*3 conv to alleviate this issue.

It is very likely that we will apply our model to high-resolution generation in the near future.
